src/app/components/file-tagging-modal/file-tagging-modal.component.html

```diff
@@ -0,0 +1,53 @@
+<!--
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+
+  SPDX-License-Identifier: EPL-2.0
+
+  Copyright Contributors to the Zowe Project.
+-->
+<zlux-tab-trap></zlux-tab-trap>
+<i class="{{icon}} modal-icon"></i>
+<h2 mat-dialog-title class="modal-mat-header">{{name}} - File Tagging</h2>
+<div class="modal-row">
+  <div class="modal-column-full-width">
+    <mat-list-item>
+      <ng-container *ngIf="isDirectory else file">Tag all files as</ng-container>
```

Comment:
Personal preference: might be better UX to add an *ngIf here for "Tag all files and subdirectories as"
when recursive is selected instead of a pop-up (or at least make pop-up font a little larger)

Reply:
I think "Recursive" is enough. "Tag all files and subdirectories as" could take more room than "Tag all files as" and break the layout of the dialog.